### PR TITLE
EWL-5294 category page article stub organism

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-5294 SG2 | Create "Category Page Article Stub Four-Up/Three-Up" Organism (Cat/Subcat)](https://issues.ama-assn.org/browse/EWL-5294)

## Description
Create category page article stub with four up on desktop, two up on tablet and full size on mobile

## To Test
- [x] `gulp serve`
- [x] visit http://localhost:3000/?p=organisms-category-page-article-stub
- [x] observe the "Content Grouping Header" atom" on top of the article stubs
- [x] observe 4 article stubs in a row with images, subcategory headers and headers
- [x] resize your viewport to tablet size
- [x] observe the article stubs stack in two columns
- [x] resize your browser to phone size
- [x] observe the article stubs stack in one column (note: the change here was approved by the UX team)
- [x] Did you test in IE 11? yes

## Visual Regressions

Add category page article stub organism test
Note: The article stub for category page has been altered. The left and right padding has been removed

## Relevant Screenshots/GIFs
screen shot 2018-06-22 at 10 07 59 am
screen shot 2018-06-22 at 10 07 50 am
screen shot 2018-06-22 at 10 07 43 am

## Remaining Tasks
N/A

## Additional Notes
N/A
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

File: styleguide/source/_patterns/03-organisms/category-page-article-stub.twig [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="ama__category-page-article-stub">
+  {% set contentGroupingHeader = categoryArticleStub.contentGroupingHeader %}
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/category-page-article-stub.twig]: This should be optional

File: styleguide/source/_patterns/03-organisms/category-page-article-stub.json [hunk 2]
```diff
@@ -0,0 +1,109 @@
+{
+  "categoryArticleStub": {
+    "contentGroupingHeader": {
+      "text": "Content grouping header"
+    },
+    "articleStubs": [
```
Comment (labbydev) [inline styleguide/source/_patterns/03-organisms/category-page-article-stub.json:6]: I'd like to see at least one example here that includes the metadata if not more.

File: styleguide/source/assets/scss/03-organisms/_category-page-article-stub.scss [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    }
+  }
+
+  .ama__article-stub {
+    @include gutter($margin-left-full...);
+    @include gutter($margin-bottom-full...);
+    width: 100%;
+
+    @include breakpoint($bp-small min-width) {
+      width: 47%;
```
Comment (labbydev) [inline styleguide/source/assets/scss/03-organisms/_category-page-article-stub.scss:22]: According to the zeplin files these should still be 4-up on tablet. 
screen shot 2018-06-20 at 6 20 18 pm

**Review by labbydev — CHANGES_REQUESTED** [review]
It looks like some of the responsive styling doesn't match up with the Zeplin files. Specifically, they should still be 4-up on tablet. Also the images should be on the left on mobile instead of on top.

screen shot 2018-06-20 at 6 20 07 pm

File: styleguide/source/_patterns/03-organisms/category-page-article-stub.twig (hunk 1 above)
Comment (designerneil) [reply styleguide/source/_patterns/03-organisms/category-page-article-stub.twig]: @labbydev during my UX collab session I asked them about the left and right and convinced them not to change it on mobile devices

Comment (designerneil) [reply styleguide/source/_patterns/03-organisms/category-page-article-stub.twig]: I'll show Toph the current implementation and see if he approves.

**Comment by designerneil** [comment]
@labbydev 

- the the metadata has been added
- the content grouping header is now optional
- as the for the amount of tiles that show. UX is aware of the implementation in this PR. They have agreed the number of tiles showing should depend on the size of the container and not a fixed number.

**Review by labbydev — APPROVED** [review]

**Pull request merged**